packages/components/atoms/Button/Button.modules.scss

```diff
@@ -1,45 +1,90 @@
+@import '../../../stylesheets/base/variables';
+@import '../../../stylesheets/base/colors';
```

Comment:
How about we import the colors file into the variables file and then we only need to import variables in the components?

Reply:
Yeah, I thought about that but was waiting for the ticket where you will include the base styles. Nevertheles I can include it in the variables file

Reply:
Yeah we can do it in that ticket no biggie